src/components/VideoPlayer.js

```diff
@@ -5,7 +5,11 @@ export default function VideoPlayer({children, video, youtube}){
 
     if(youtube == "true"){
         return (
+            <>
             <iframe className="border border-rounded m-3" width="100%" height="450" src={video} title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+            {children}
```

Comment:
These changes are already there in the master, can you please resolve this, by removing them?